example/adapters/sequelize.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * This is a very rough-edged example, the idea is to still work with the fact that oidc-provider
+ * has a rather "dynamic" schema. This example uses sequelize with sqlite, and all dynamic data
+ * uses JSON fields. id is set to be the primary key, grantId should be additionaly indexed for
+ * models where these fields are set.
```

Comment:
additionally mention the userCode index for the DeviceCode model and uid index for Session

Reply:
Updated